File: backend/ws-service/src/main/java/com/example/wsservice/chat/dto/ReactionSummaryDto.java
package com.example.wsservice.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.List;

/** chat-service 의 집계와 같은 모양. 그대로 앱에 내려간다. */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReactionSummaryDto implements Serializable {
    private String emoji;
    private int count;
    private List<String> userIds;
}

File: backend/ws-service/src/main/kotlin/com/example/wsservice/chat/dto/ReactionSummaryDto.kt
package com.example.wsservice.chat.dto

import java.io.Serializable

/** chat-service 의 집계와 같은 모양. 그대로 앱에 내려간다. */
data class ReactionSummaryDto(
    var emoji: String? = null,
    var count: Int = 0,
    var userIds: List<String>? = null,
) : Serializable
